mdal/mdal_logger.hpp

```diff
@@ -0,0 +1,49 @@
+/*
+ MDAL - Mesh Data Abstraction Library (MIT License)
+ Copyright (C) 2020 Tomas Mizera (tomas.mizera2 at gmail dot com)
+*/
+
+#ifndef MDAL_LOGGER_H
+#define MDAL_LOGGER_H
+
+#include <string>
+
+#include "mdal.h"
+
+namespace MDAL
+{
+  /**
+   * Namespace including functions responsible for handling logs.
+   *
+   * Use in code as: MDAL::Log::error/warning( MDAL_Status, logMessage ).
+   * By default, output from logger is set to standard stdout, but it is
+   * possible to set custom logger output with function setLoggerCallback.
+   */
+  namespace Log
+  {
+    struct LogStruct
```

Comment:
Move to mdal_utils.hpp and rename to `MDAL::Error( MDAL_Status s, const std::string& m, const std::string& m, const std::string& driver = "" )`